### PR TITLE
Update both status and custom resource

### PR DESCRIPTION
**Pull request by csviri**

### CODE REVIEW

File: operator-framework/src/main/java/io/javaoperatorsdk/operator/api/UpdateControl.java [hunk 1]
```diff
@@ -27,6 +27,15 @@ private UpdateControl(
     return new UpdateControl<>(customResource, true, false);
   }
 
+  /**
+   * As a results of this there will be two call to K8S API. First the custom resource will be
+   * updates then the status sub-resource.
+   */
+  public static <T extends CustomResource>
```
Comment (psycho-ir) [inline operator-framework/src/main/java/io/javaoperatorsdk/operator/api/UpdateControl.java:34]: how about `updateCustomResourceAndStatus`?

Comment (csviri) [reply operator-framework/src/main/java/io/javaoperatorsdk/operator/api/UpdateControl.java:34]: hmm, makes sense will change it!! thx

File: operator-framework/src/test/java/io/javaoperatorsdk/operator/doubleupdate/subresource/DoubleUpdateTestCustomResource.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    return status;
+  }
+
+  public void setStatus(DoubleUpdateTestCustomResourceStatus status) {
+    this.status = status;
+  }
+
+  @Override
+  public String toString() {
+    return "TestCustomResource{"
```
Comment (metacosm) [inline operator-framework/src/test/java/io/javaoperatorsdk/operator/doubleupdate/subresource/DoubleUpdateTestCustomResource.java]: Copy&Paste error… 😄 
```suggestion
    return "DoubleUpdateTestCustomResource{"
```

**Review by metacosm — APPROVED** [review]
LGTM

File: operator-framework/src/test/java/io/javaoperatorsdk/operator/doubleupdate/DoubleUpdateTestCustomResource.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    return status;
+  }
+
+  public void setStatus(DoubleUpdateTestCustomResourceStatus status) {
+    this.status = status;
+  }
+
+  @Override
+  public String toString() {
+    return "TestCustomResource{"
```
Comment (metacosm) [inline operator-framework/src/test/java/io/javaoperatorsdk/operator/doubleupdate/DoubleUpdateTestCustomResource.java:29]:
```suggestion
    return "DoubleUpdateTestCustomResource{"
```

**Pull request merged**